mk/xe-linux-distribution-test

```diff
@@ -53,6 +53,17 @@ do_test()
 
 do_test "testcases/debian-sid"           "debian"
 do_test "testcases/debian-10.6"          "debian 10 6"
+do_test "testcases/fc3"                  "fedora 3 0"
+do_test "testcases/ddk-0.5.6-2991c"      "xe-ddk 0.5.6 2991c"
+do_test "testcases/sdk-0.5.6-2991c"      "xe-sdk 0.5.6 2991c"
+do_test "testcases/sle-15"               "sles 15 0"
+do_test "testcases/sle-15sp1"            "sles 15 1"
+do_test "testcases/centos7"              "centos 7 0"
+do_test "testcases/centos-stream-8"      "centos 8 0"
+do_test "testcases/rocky-8.3"            "rocky 8 3"
+do_test "testcases/almalinux-8.3"        "almalinux 8 3"
+do_test "testcases/slackware-15.0"       "slackware 15 0"
+# EOL distros
```

Comment:
Please add a line above to separate the "EOL distributions test cases" and the "Non EOL distributions test cases"

Reply:
Also change EOL distros to "EOL distributions test cases"